server/app/controllers/applicant/ApplicantProgramBlocksController.java

```diff
@@ -165,7 +166,22 @@ public CompletionStage<Result> confirmAddress(
     ImmutableList<AddressSuggestion> suggestions =
         addressSuggestionJsonSerializer.deserialize(
             maybeAddressJson.orElseThrow(() -> new RuntimeException("Address JSON missing")));
+    return confirmAddressWithSuggestions(
+        request, applicantId, programId, blockId, inReview, selectedAddress, suggestions);
+  }
 
+  /**
+   * This method saves the selected corrected address to the db and redirects the user to the next
+   * screen
+   */
```

Comment:
**nitpick:** update comment wording and formatting to match style guide: https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment